themes/delphi/layouts/partials/footer.html

```diff
@@ -9,7 +9,7 @@ <h5 class="uk-text-bold">COVIDcast</h5>
         <ul class="uk-list">
```

Comment:
The h5 element above this still refers to COVIDcast, should probably be Epidemic Signals